src/ArticleContent.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import axios from "axios";
+
+export function ArticleContent() {
+    const { article_id } = useParams();
+    const [articleContent, setArticleContent] = useState({});
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        axios
+            .get(`https://xqnews.onrender.com/api/articles/${article_id}`)
```

Comment:
Would be better if all the api calls can be abstracted to one file and just call the function from the component instead of write a new one here. Making api call function can make it reusable